Pecan PIE doc updates. (#30)
stjude/sjcloud-docs: docs/guides/tools/pecan-pie.md

## Advanced options

> The "Advanced option" panel lets you customize the behavior of th

Reviewer: ```suggestion
The "Advanced option" panel lets you customize the behavior of the
```

pipeline:

> * Gene list: Pick a gene list from the pulldown, this filters your

Reviewer: ```suggestion
* Gene list: Pick a gene list from the pulldown. This filters your
```

  variants to genes in the specified list. This option is required and turned

>   on automatically if your uploaded file is 2 megabyte or larger. See the

Reviewer: ```suggestion
  on automatically if your uploaded file is 2 megabytes or larger. See the
```

  [frequently asked questions](#frequently-asked-questions) for more
  information. This option reduces the variant processing burden on PIE by
  removing variants that will not be assigned a medal in any case because
  they are not on the cancer predisposition gene list. You can review the
  genes by clicking on the link that will appear just below the pull down
  titled "See gene list".
* Custom gene list: Choosing "custom" as your gene list will open a
  window that will let you paste in a list of genes. Any invalid genes will

>   be dropped from your list automatically. You can separate your genes spaces

Reviewer: ```suggestion
  be dropped from your list automatically. You can separate your genes by spaces
```

**Q: Are there limits on the size of VCF files?**

Uploaded files must not exceed 4 gigabytes. If an uploaded file is larger

> than 2 megabyte, cancer predisoposition gene list filter will be

Reviewer: ```suggestion
than 2 megabytes, the cancer predisposition gene list filter will be
```
